docs/howto/account-billing/forgot-your-password.md

```diff
@@ -0,0 +1,55 @@
+# Resetting your password or reclaiming your username
+
+Whether you have to reset a forgotten password or remember the usernames associated with your {{brand}} account, you can do so via the {{gui}}.
+
+## Reset lost password
+
+Begin by navigating to the {{gui}}. At the bottom left-hand side of the page, click on the "Forgot your password?" button.
```

Comment:
Future changes/reviews will be greatly facilitated by using one line per sentence, like so:

```suggestion
Begin by navigating to the {{gui}}.
At the bottom left-hand side of the page, click on the "Forgot your password?" button.
```